cli/require-protobufjs.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = function requireProtobufjs() {
+    try {
+        // local development, i.e. forked from github
+        return require("..");
+    } catch (e) {
+        // installed as a peer dependency
+        try {
+            return require("protobufjs-taylorm");
```

Comment:
Seems like `-taylorm` is not supposed to be here (and below). Looks like your test package name made its way here :)

Reply:
oh crap. fixing that now

Reply:
fixed